test/Dapr.Actors.Test/ActorIdTests.cs

```diff
@@ -117,7 +117,7 @@ public void Initialize_New_ActorId_Object_With_Null_Or_Whitespace_Id(string id)
         {
             Assert.Throws<ArgumentException>(() =>
             {
-                ActorId actorId = new ActorId(id);
+                new ActorId(id);
             });
```

Comment:
This can be simplified by simply doing

```csharp 
Assert.Throws<ArgumentException>(() => new ActorId(id));
```

Reply:
Thank you! Addressed in my latest commit!